docs/enable-libraries.md

```diff
@@ -0,0 +1,40 @@
+[../README.md#Guides](../README.md#guides)
+
+# Enable the New Architecture for Libraries
+
+**The first step for supporting the New Architecture in your library is to ensure that it is compatible with the [interop layer](https://github.com/reactwg/react-native-new-architecture/discussions/135)**. The interop layer makes it possible to use existing native modules written for the legacy architecture with the New Architecture. **You should do this before proceeding to convert your library to natively use TurboModules/Fabric.** Starting with React Native 0.74, the interop layer is enabled by default and, if your library is compatible, then no changes will be needed on the user's side in order to use your library.
+
+## Using the interop layer
+
+The interop layer will generally work out of the box with simple libraries, but the more complex your library is, the more likely it is that you will need to make some changes. The following sections will guide you through the process of verifying that your library works with the interop layer and fixing common issues that you encounter.
+
+> [!IMPORTANT]
+> If you have already added a Codegen spec to your library, but the library is not fully converted to TurboModules/Fabric, we recommend that you delete it before proceeding. You can add this back when you are ready to convert your library to natively use TurboModules/Fabric without the interop layer.
+
+### Test your library
+
+Follow [the guide for testing your library against the latest version of React Native](https://gist.github.com/cipolleschi/82b7a9561b8861330efabbd3eb08c6f5). Be sure to test all of the functionality of your library to ensure that it works as expected.
+
+### Common issues (JavaScript)
+
+(list them here, along with solutions and links to pull requests)
+
+### Common issues (iOS)
+
+(list them here, along with solutions and links to pull requests)
+
+### Common issues (Android)
+
+(list them here, along with solutions and links to pull requests)
+
```

Comment:
@Kudo @alanjhughes @gabrieldonadel - can you add some suggestions here for common issues to seed this with some initial useful tips? we can expand on it later. could base it off of the prs you've had to open recently / the "common problems" tab on spreadsheet.